data-search-widget/data-search-widget.js

```diff
@@ -697,7 +698,7 @@
             //when reset
             if (config.maps) {
               clearMarkers();
-              addMarkers(globalData);
+              addMarkers(data);
```

Comment:
```suggestion
              addMarkers(data, true);
```
Could we add an optional `updateMapBounds` argument to `addMarkers()` function? Further details below. 
Passing in 'reset' here will allow us to update the map bounds to show all results again.

Reply:
Thanks Jordan. Good pick up. I added `updateMapBounds` to `addMarker` function. However I have to call it with value of `false` when reset is hit, call with `true` otherwise.

Reply:
Resolved in updates 👍